client/transactions/list/index.tsx

```diff
@@ -605,6 +605,7 @@ export const TransactionsList = (
 
 	const endpointExport = async ( language: string ) => {
 		// We destructure page and path to get the right params.
+		// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
We might need to restore the linter suppression on line 717 as well.

Reply:
Great catch! Restored.